bandit/blacklists/calls.py

```diff
@@ -569,4 +569,65 @@ def gen_blacklist():
         'attacks. Consider using tmpfile() instead.'
         ))
 
+    # updated rulesets starts here
+
+    sets.append(utils.build_conf_dict(
+        'yaml_load', 'B326',
+        ['yaml.load'],
+        """
+        Use of this deprecated deserialize method leads to
+        remote code execution. See CVE-2017-18342.
+        """
+        'Use yaml.safe_load instead.'
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'response_splitting', 'B327',
+        [
+            'urllib2.Request.add_header',
+            'urllib.URLopener.addheader',
+            'urllib.URLopener.addheaders'
+        ],
+        'Python Urllib and Urllib2 are vulnerable to CRLF injection.'
+        'See https://bugs.python.org/issue36276'
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'Base64_encoding_used', 'B328',
+        [
+            'base64.b64encode',
+            'base64.standard_b64encode',
+            'base64.urlsafe_b64encode',
+            'base64.b16encode',
+            'base64.b32encode',
+            'base64.encodestring',
+            'flask_scrypt.enbase64',
+            'base64.b64decode'
+        ],
+        """
+        Weak encoding alogirthm. Encoding and storing sensitive
+        information using this algorithm gives a false sense of security.
+        """
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'Server_Side_Template_Injection', 'B329',
+        ['render_template_string', 'render_template'],
+        """
+        User controlled Values injected into the template can lead to
+        arbitrary code executions in the server. Check if input values
+        are properly sanitized. See https://blog.nvisium.com/p263
+        """
+        ))
+
+    sets.append(utils.build_conf_dict(
+        'file_upload', 'B330',
+        ['Flask.send_file'],
```

Comment:
This would probably be better as a plugin which can do more advanced checking on the usage such as the mime type you mentioned.